content/html/power-request-hosting.html

```diff
@@ -107,25 +96,17 @@
         name="other_information" cols="60" rows="5" class="form-textarea"></textarea></div>
         <div class="description">Is there anything additional you would like to provide for your request?</div>
       </div>
-      <h4>POWER CI Access Question(s)</h4>
-      <div class="form-item webform-component webform-component-textfield" id="webform-component-ci-github">
-        <label for="edit-submitted-ci-github">GitHub Username(s) for POWER CI</label>
-        <input type="text" id="edit-submitted-ci-github" name="ci-github" value="" size="60" maxlength="128"
-        class="form-text" />
-        <div class="description">We use Github OAuth to authenticate into the POWER CI system. If you selected "Yes"
-        above, please provide a comma separated list of Github username(s) to gain access.</div>
-      </div>
       <h4>OpenStack Access Question(s)</h4>
       <div class="form-item webform-component webform-component-select" id="webform-component-power-architecture">
-        <label for="edit-submitted-power-architecture">POWER Architecture <span class="form-required" title="This field
-        is required.">*</span></label>
+        <label for="edit-submitted-power-architecture">POWER Architecture <span class="form-required"
+            title="This field is required.">*</span></label>
         <select id="edit-submitted-power-architecture" name="power_architecture" class="form-select required">
           <option value="ppc64le POWER (Little Endian)" selected="selected">ppc64le POWER (Little Endian)</option>
```

Comment:
@ramereth Is this still something we want, seeing as `ppc64le` is the only that will be used in this form?